APMrover2/sailboat.h

```diff
@@ -24,7 +24,8 @@ class Sailboat
     static const struct AP_Param::GroupInfo var_info[];
 
     // enabled
-    bool enabled() const { return enable != 0;}
+    bool sail_enabled() const { return enable != 0;}
+    bool nav_enabled();
```

Comment:
could we add a comment above "nav_enabled()" to clarify?  does it actually mean "tacking_enabled" perhaps?

Reply:
added a small description, currently this is just if we are tacking upwind, it may become more if/when i get this drag polar stuff working